examples/gno.land/r/gnoland/users/v1/render.gno

```diff
@@ -0,0 +1,146 @@
+package v1
+
+import (
+	"std"
+	"strconv"
+
+	"gno.land/p/demo/avl/pager"
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/moul/md"
+	"gno.land/p/moul/realmpath"
+	"gno.land/p/moul/txlink"
+
+	"gno.land/r/demo/profile"
+	"gno.land/r/sys/users"
+)
+
+var (
+	connectUrlBase       = "https://gno.studio/connect/view/gno.land/r/gnoland/users/v1?network=" + std.GetChainID()
+	connectRegisterUrl   = connectUrlBase + "#Register"
+	connectUpdateNameUrl = connectUrlBase + "#UpdateName"
+)
+
+func Render(path string) string {
+	req := realmpath.Parse(path)
+
+	if req.Path == "" {
+		return renderHomePage(path)
+	}
+
+	// Otherwise, render the user page
+	return renderUserPage(req.Path)
+}
+
+func renderHomePage(path string) string {
+	var out string
+
+	out += "# gno.land user registry\n"
+
+	if paused {
+		out += md.HorizontalRule()
+		out += md.H2("This realm is paused.")
+		out += md.Paragraph("Check out [`gno.land/r/gnoland/users`](/r/gnoland/users) for newer versions of the registry.")
```

Comment:
This makes me think that `md.Paragraph` could include a built-in string replacement feature, allowing this to be done as follows:

```go
		out += md.Paragraph("Check out {{link}} for newer versions of the registry.",
			"link", md.Link(),
		)
```

(This is not related to this PR.)